modin/core/execution/ray/implementations/pandas_on_ray/partitioning/virtual_partition.py

```diff
@@ -423,6 +423,10 @@ def drain_call_queue(self, num_splits=None):
         num_splits : int, default: None
             The number of times to split the result object.
         """
+        if len(self.call_queue) == 0:
+            # this implicitly calls `drain_call_queue`
+            _ = self.list_of_blocks
```

Comment:
Why do we need to call this even if the call queue is empty?

Reply:
Partitions inside a virtual one can have a non-zero queue.

Reply:
makes sense